ext/nnstreamer/tensor_filter/tensor_filter_edgetpu.cc

```diff
@@ -43,10 +44,33 @@ namespace tensorfilter_edgetpu {
 void _init_filter_edgetpu (void) __attribute__ ((constructor));
 void _fini_filter_edgetpu (void) __attribute__ ((destructor));
 
+
+enum class edgetpu_subplugin_device_type : uint32_t {
+  USB = static_cast<uint32_t>(edgetpu::DeviceType::kApexUsb),
+  PCI = static_cast<uint32_t>(edgetpu::DeviceType::kApexPci),
+  DUMMY = 99,
+};
+
+static const std::string edgetpu_subplugin_device_type_name (
+    edgetpu_subplugin_device_type t)
+{
+    switch(t)
+    {
+        case edgetpu_subplugin_device_type::PCI:
+          return "pci";
+        case edgetpu_subplugin_device_type::DUMMY:
+          return "dummy";
+        case edgetpu_subplugin_device_type::USB:
+        default:
+          return "usb";
```

Comment:
IMO, it's natural to return `dummy` in default case. Is there any implied purpose on this?

Reply:
@dongju-chae This is a sub-plugin for EdgeTPU so that it is natural to work with the device by default for me. The "dummy" mode is only for the testing.

Reply:
Got it